src/main/java/fi/fmi/avi/model/SpaceWeatherAdvisory/SpaceWeatherAdvisoryAnalysis.java

```diff
@@ -0,0 +1,22 @@
+package fi.fmi.avi.model.SpaceWeatherAdvisory;
+
+import java.util.List;
+import java.util.Optional;
+
+import fi.fmi.avi.model.PartialOrCompleteTimeInstant;
+
+public interface SpaceWeatherAdvisoryAnalysis {
+    PartialOrCompleteTimeInstant getTime();
+
+    Optional<Type> getAnalysisType();
+
+    Optional<List<SpaceWeatherRegion>> getRegion();
```

Comment:
Perhaps we should use `List<SpaceWeatherRegion>` without `Optional` here. I'm not aware that an empty list would be semantically different from a non-existent list. E.g. IWXXM 3.0 schema requires at least one `region` (which may be `nil`).